apps/amiapp_flutter/lib/firebase_options.dart

```diff
@@ -0,0 +1,69 @@
+// File generated by FlutterFire CLI.
+// ignore_for_file: lines_longer_than_80_chars, avoid_classes_with_only_static_members
+import 'package:firebase_core/firebase_core.dart' show FirebaseOptions;
+import 'package:flutter/foundation.dart'
+    show defaultTargetPlatform, kIsWeb, TargetPlatform;
+
+/// Default [FirebaseOptions] for use with your Firebase apps.
+///
+/// Example:
+/// ```dart
+/// import 'firebase_options.dart';
+/// // ...
+/// await Firebase.initializeApp(
+///   options: DefaultFirebaseOptions.currentPlatform,
+/// );
+/// ```
+class DefaultFirebaseOptions {
+  static FirebaseOptions get currentPlatform {
+    if (kIsWeb) {
+      throw UnsupportedError(
+        'DefaultFirebaseOptions have not been configured for web - '
+        'you can reconfigure this by running the FlutterFire CLI again.',
+      );
+    }
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.android:
+        return android;
+      case TargetPlatform.iOS:
+        return ios;
+      case TargetPlatform.macOS:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for macos - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      case TargetPlatform.windows:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for windows - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      case TargetPlatform.linux:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for linux - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      default:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions are not supported for this platform.',
+        );
+    }
+  }
+
+  static const FirebaseOptions android = FirebaseOptions(
+    apiKey: 'AIzaSyASGdGzS1zfpzNdZTXtthSWEHs3b0mmOGA',
+    appId: '1:1001564516023:android:ceccfaa92480cba51a2732',
+    messagingSenderId: '1001564516023',
+    projectId: 'remote-habits',
+    storageBucket: 'remote-habits.appspot.com',
+  );
+
+  static const FirebaseOptions ios = FirebaseOptions(
```

Comment:
are all of these values meant to be public?

Reply:
According [to the flutter-fire docs](https://firebase.google.com/docs/flutter/setup?platform=ios) for "What does this` flutterfire configure` workflow do?", it appears that this file is similar to `google-services` files that all our FCM sample apps contain. 

![CleanShot 2023-12-18 at 11 51 59@2x](https://github.com/customerio/customerio-flutter/assets/2041082/ecde0465-10b4-49e3-921a-cd6b5582da75)